data/json/effects_on_condition/eoc_riddles.json

```diff
@@ -0,0 +1,391 @@
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_XE_ED_RIDDLE_5",
+    "effect": [
+      {
+        "run_eoc_selector": [ "EOC_XE_GIVE_CLASS_DREAMER", "EOC_XE_GIVE_CLASS_EATER" ],
+        "title": "Leader or Follower?",
+        "names": [ "Leader", "Follower" ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_XE_GIVE_CLASS_EATER",
+    "effect": [ { "u_add_trait": "EATER" }, { "run_eocs": [ "EOC_XE_RUN_GIVE_CLASS_1_AFTERMATCH" ] } ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_XE_GIVE_CLASS_DREAMER",
+    "effect": [ { "u_add_trait": "DREAMER" }, { "run_eocs": [ "EOC_XE_RUN_GIVE_CLASS_1_AFTERMATCH" ] } ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_XE_RUN_GIVE_CLASS_1_AFTERMATCH",
+    "effect": [
+      {
+        "run_eoc_selector": [ "EOC_NONE", "EOC_NONE", "EOC_NONE", "EOC_NONE", "EOC_NONE", "EOC_NONE", "EOC_NONE", "EOC_NONE" ],
+        "title": "It doesn't matter…",
+        "names": [
+          "It doesn't matter…",
+          "It doesn't matter…",
+          "It doesn't matter…",
+          "It doesn't matter…",
+          "It doesn't matter…",
+          "It doesn't matter…",
+          "It doesn't matter…",
+          "It doesn't matter…"
+        ]
+      },
+      { "u_message": "Since your kind is doomed…", "type": "mixed" },
+      { "math": [ "magic_potential", "=", "0" ] }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_XE_RUN_RIDDLE_INVENTOR_DREAMSMITH",
+    "effect": {
+      "weighted_list_eocs": [
+        [ "EOC_XE_ID_RIDDLE_1", { "const": 1 } ],
+        [ "EOC_XE_ID_RIDDLE_2", { "const": 1 } ],
+        [ "EOC_XE_ID_RIDDLE_3", { "const": 1 } ],
+        [ "EOC_XE_ID_RIDDLE_4", { "const": 1 } ],
+        [ "EOC_XE_ID_RIDDLE_5", { "const": 1 } ]
+      ]
+    }
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_XE_ID_RIDDLE_1",
+    "effect": [
+      {
+        "run_eoc_selector": [ "EOC_XE_GIVE_CLASS_INVENTOR", "EOC_XE_GIVE_CLASS_DREAMSMITH" ],
+        "title": "Wisdom or Foolishness?",
+        "names": [ "Wisdom", "Foolishness" ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_XE_ID_RIDDLE_2",
+    "effect": [
+      {
+        "run_eoc_selector": [ "EOC_XE_GIVE_CLASS_DREAMSMITH", "EOC_XE_GIVE_CLASS_INVENTOR" ],
+        "title": "Purity or Contamination?",
+        "names": [ "Purity", "Contamination" ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_XE_ID_RIDDLE_3",
+    "effect": [
+      {
+        "run_eoc_selector": [ "EOC_XE_GIVE_CLASS_DREAMSMITH", "EOC_XE_GIVE_CLASS_INVENTOR" ],
+        "title": "Past or Future?",
+        "names": [ "Past", "Future" ]
+      }
+    ]
+  },
+  {
+    "type": "effect_on_condition",
+    "id": "EOC_XE_ID_RIDDLE_4",
+    "effect": [
+      {
+        "run_eoc_selector": [ "EOC_XE_GIVE_CLASS_INVENTOR", "EOC_XE_GIVE_CLASS_DREAMSMITH" ],
+        "title": "Hard or Soft?",
```

Comment:
complex or straightforward?

Just a thought that since dreamsmith works in metals it might seem counter to that to call it soft.